### ISSUE TITLE
Can't reproduce results on VisDA-2017

### ISSUE THREAD

**Issue by CASlzwde** [kgl-prml/Contrastive-Adaptation-Network-for-Unsupervised-Domain-Adaptation#14]
the original paper's result is 87.2, but my average result (the best value of 10 experiment results) is 86.6.

**Comment by ningding-o** [comment]
Yeah, the same situation also happened to me.  I got an average result of 86.5%. 
Guess the author might have applied some tricks during training process

**Comment by ygjwd12345** [comment]
Test at (loop 49, iters: 25399) with mean_accu: 86.8898.
I think the result is acceptable.
